container/run.sh

```diff
@@ -40,6 +40,8 @@ ENVIRONMENT_VARIABLES=
 REMAINING_ARGS=
 INTERACTIVE=
 USE_NIXL_GDS=
+RUNTIME=nvidia
```

Comment:
What's the use case to introduce the `--runtime` flag? Does `--gpus none` not work for cpu only case?